device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -72,9 +72,7 @@
     private final Set<ClientConfiguration> clientConfigurations;
     private IotHubListener listener;
     private TransportException savedException;
-    private boolean reconnectionScheduled = false;
     private final Object executorServiceLock = new Object();
-    private final Map<String, Boolean> reconnectionsScheduled = new ConcurrentHashMap<>();
```

Comment:
We used this map to keep track of which device sessions had already started reconnection so that we could not spawn two threads to reconnect the same device. Now, notifying the transport layer of a disconnection can be done multiple times without any negative impact so it doesn't make sense to track this anymore